katalog/elasticsearch-single/elasticsearch.yml

```diff
@@ -90,7 +90,7 @@ spec:
         - name: elasticsearch
           image: docker.elastic.co/elasticsearch/elasticsearch
           env:
-            - name: http.host
+            - name: network.host
               value: "0.0.0.0"
             - name: ES_JAVA_OPTS
               value: -Xms2G -Xmx2G
```

Comment:
I would add the following comment here: 
# Java heap must be 50% of total ram limits and min/max must be equal to avoid heap runtime resizing (a very costly process)

Reply:
Currently, we are specifying to have the top 3Gi of ram. 
Do you think its a good idea to change the limit to 4Gi leaving the Xms and Xmx to 2Gi?

Reply:
Or maybe I can specify less Xms Xmx (1.5) with the current limit of 3Gi

Reply:
✅